### DISCUSSION TITLE
Cant find dbn_pipeline file

### DISCUSSION THREAD

**Discussion by web1havv** [akdasUAF/ForestFireDetection#26] [Q&A] [answered]
Respected Mentor
I am unable to find the file dbn_pipeline_model.joblib.gz that we were asked to move to models folder. I am unable to find it on Google drive folder as well (only two files are there, three files are mentioned in readme markdown.) Can you please help me out with that?
Regards
Vaibhav

**Comment by YaliWang2019** [comment, marked as answer]
> Respected Mentor I am unable to find the file dbn_pipeline_model.joblib.gz that we were asked to move to models folder. I am unable to find it on Google drive folder as well (only two files are there, three files are mentioned in readme markdown.) Can you please help me out with that? Regards Vaibhav

Hi Vaibhav, the extracted joblib file was already in this directory: Models/weights, I'll update the instructions in the Readme file. Sorry for the confusion.
Best,
Yali

**Reply by web1havv** [reply]
Thanks for the clarification.
